kubernetes/proxy.go

```diff
@@ -24,6 +24,7 @@ const selfServicePort = 80
 
 func StartProxy(kubernetesProvider *Provider, proxyHost string, srcPort uint16, selfNamespace string, selfServiceName string) (*http.Server, error) {
 	log.Info().
+		Str("proxyHost", proxyHost).
```

Comment:
```suggestion
		Str("proxy-host", proxyHost).
```

We use `kebab-case` style for debugging values in `log` calls project-wide.